src/include/attribute.h

```diff
@@ -192,7 +192,7 @@ typedef struct attribute attribute;
 struct attribute_def {
 	char	*at_name;
 	int	(*at_decode)(attribute *patr, char *name, char *rn, char *val);
-	int	(*at_encode)(attribute *pattr, pbs_list_head *phead,
+	int	(*at_encode)(const attribute *pattr, pbs_list_head *phead,
```

Comment:
Please put all argument in one line (applies to other files also)

Reply:
it'll become a very long line, i think breaking is more readable than scrolling left-right in your IDE to see all parameters. So, I'd like to leave it as is for now. I will attempt to reduce the number of parameters for functions which take too many in future refactoring PRs.